src/main/java/org/cancogenvirusseq/muse/service/SubmissionService.java

```diff
@@ -168,12 +164,17 @@
         .map(filePart -> Tuples.of(mapEntry.getKey(), filePart));
   }
 
-  private static Flux<Tuple2<String, String>> readFileContentToString(
+  private static Flux<SubmissionFile> readFileContentToString(
```

Comment:
yeah Tuple kinda suck the more I use them ... what we really want are quick types like Typescript has but I guess these little inner classes will do